spec/features/staff_can_create_a_property_defect_spec.rb

```diff
@@ -65,6 +67,10 @@
       expect(page).to have_content('Property')
       expect(page).to have_content(property.address)
     end
+
+    within('.evidence') do
```

Comment:
Can we check the evidence has been uploaded here too?

Reply:
Whoops, of course, thanks! 😄